iRonCub-Mk1_1/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(appname iRonCub-Mk1_1)
+
+file(GLOB xml ${CMAKE_CURRENT_SOURCE_DIR}/*.xml)
+file(GLOB ini ${CMAKE_CURRENT_SOURCE_DIR}/*.ini)
+
+yarp_install(FILES ${xml} DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})
+yarp_install(FILES ${ini} DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})
+yarp_install(DIRECTORY calibrators DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})  
+yarp_install(DIRECTORY cartesian DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})
+yarp_install(DIRECTORY wrappers DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})
+yarp_install(DIRECTORY hardware DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname})  
+yarp_install(DIRECTORY camera DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname}) 
+yarp_install(DIRECTORY estimators DESTINATION ${ICUBCONTRIB_ROBOTS_INSTALL_DIR}/${appname}) 
```

Comment:
What happens if we duplicate only this part for `iRonCub-Mk1` and `iRonCub-Mk1_1`?

Reply:
Notice that the install folder is different. It appends ``${appname}`` at the end of the path, and you set it to ``iRonCub-Mk1_1``. Hence, if the ``appname`` is different, it will create different folders. So I guess it should be ok.